packages/lesswrong/lib/routes.ts

```diff
@@ -1597,6 +1597,16 @@ addRoute(
   }
 );
 
+addRoute(
+  {
+    name: 'curationQueue',
+    path:'/curationQueue',
+    componentName: "CurationPage",
```

Comment:
It's somewhat better to have the route match the componentName, all else being equal.

Reply:
Changed to /curation